x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_schema_editor/index.tsx

```diff
@@ -86,6 +96,20 @@ export const StreamDetailSchemaEditor = ({ definition, refreshDefinition }: Sche
 
   return (
     <EuiFlexGroup direction="column" gutterSize="none" css={{ height: '100%' }}>
+      {isRootStream && (
+        <>
+          <EuiCallOut
+            iconType="info"
+            title={i18n.translate('xpack.streams.schemaEditor.rootStreamReadOnlyMode', {
+              defaultMessage:
+                'Root streams are selectively immutable and their schema cannot be modified. To enrich data, partition a new child stream and add processors to it.',
```

Comment:
since this is the Schema page, should we maybe change this text to the following? 

`Root streams are selectively immutable and their schema cannot be modified. To change the schema or add processing steps, partition a new child stream first`

Reply:
That sounds better to me, tweaked!